client/src/component/FAQ/FAQ.jsx

```diff
@@ -0,0 +1,123 @@
+
+
+import React from "react";
+import 'font-awesome/css/font-awesome.min.css'
+import './FAQ.css'
+
+
+
+const Faqs = () => (
+  <div className="Faqs" id="Faqs">
+      <h2>
+        <u>FREQUENTLY ASKED QUESTIONS</u>
+      </h2>
+      <div className="container">
+        <div className="column left">
+          <div className="question">
+            <h3>Who can attend?</h3>
+            <p>
+              All undergraduate and graduate students of any major or high
+              school students are welcome! High school students and those under
+              the age of 18 are required to fill out a{" "}
+              <a
+                href="https://risk.ucmerced.edu/sites/risk.ucmerced.edu/files/documents/waiver-voluntary.pdf"
+                target="_blank"
+                rel="noopener noreferrer"
+                style={{ color: "#fff" }}
+              >
+                liability waiver
+              </a>{" "}
+              and have it sent to{" "}
+              <a
+                href="mailto:general@hackmerced.com?subject=HackMerced Questions&body=Dear HackMerced,"
+                style={{ textDecoration: "none", color: "#D7AA27" }}
+              >
+                general@hackmerced.com
+              </a>
+            </p>
+          </div>
+          <div className="question">
+            <h3>How do teams work?</h3>
+            <p>
+              You can be in teams of up to 4 people and it is highly encouraged
+              to find at least one other person to participate with! If you need
+              help forming a team, we'll help set you up with others who are
+              looking for teammates.
+            </p>
+          </div>
+          <div className="question">
+            <h3>How much will it cost me?</h3>
+            <p>
+              Admission, food and drinks are absolutely free! However, we cannot
+              cover any travel costs.
+            </p>
+          </div>
+          <div className="question">
+            <h3>What can I build?</h3>
+            <p>
+              You and your team can build a project towards a theme of your
+              choosing!
+            </p>
+          </div>
+        </div>
+        <div className="column right">
```

Comment:
having it as `        <div className="column right">` invokes both the `column` CSS and `right` css.
```suggestion
        <div className="column-right">
```
This would give you the option to add custom column CSS to your file.
or you could make it `<div className="column">` to keep the column width.